python/perf-kernels/flash-attention.py

```diff
@@ -680,82 +793,118 @@ def _bwd_kernel_dq(
     stride_qz, stride_qh, stride_qm, stride_qk,
     stride_kz, stride_kh, stride_kn, stride_kk,
     stride_vz, stride_vh, stride_vk, stride_vn,
-    seqlen_q, seqlen_k, dropout_p, philox_seed, philox_offset_base,
+    stride_oz, stride_oh, stride_om, stride_ok,
```

Comment:
Similar to dkdv, dq kernel needs strides from do and dq (again, not strides from o)